package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mobify-platform-sdks",
-  "version": "1.5.0-alpha.1",
```

Comment:
Ops, this is wrong.
I believe if you re-run the `bump-version` script for a second time, it will fix the version here as well.